src/index.html

```diff
@@ -9,5 +9,29 @@
     <h1>Hello Push</h1>
     <img src="./assets/img/moramia.png" alt="image2">
     <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ea rem, placeat assumenda facere voluptatem amet molestias tempora reiciendis quibusdam non, eaque dolorem natus labore ut, magni facilis itaque! Blanditiis, error?</p>
+    <footer class="footer">
+        <div class="footer-content">
+            <div class="footer-image">
+                <img src="./assets/img/logo2.png" alt="logo2" class="footer-img" />
+                <p class="footer-cp">
+                    Copyright ©2020 All rights reserved | This template is made with  &#9829;  by PushDev & Re-distributed by Moramia
+                </p>
+            </div>
+            <div class="menu-wrap">
+                <input type="checkbox" class="toggler">
+                <div class="hamburger"><div class="footer-center-line"></div></div>
+                <div class="menu">
+                    <div>
```

Comment:
what's the purpose of this div? can it be removed?

Reply:
sorry I forgot to remove it on the pr